dockstore-webservice/src/main/java/io/dockstore/webservice/core/OrganisationUser.java

```diff
@@ -0,0 +1,146 @@
+package io.dockstore.webservice.core;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@Entity
+@Table(name = "OrganisationUser")
+public class OrganisationUser implements Serializable {
+
+    @EmbeddedId
+    private OrganisationUserId id;
+
+    @ManyToOne
+    @JoinColumn(name = "userId", insertable = false, updatable = false)
+    private User user;
+
+    @ManyToOne
+    @JoinColumn(name = "organisationId", insertable = false, updatable = false)
+    private Organisation organisation;
+
+    public enum Role {
+        ADMIN, MAINTAINER, MEMBER
```

Comment:
Might be able to simplify. Let's discuss permissions (I think I meant admins/curators on the site can approve all orgs and aren't stored here, the roles here shouldn't overlap site roles)